tests/show_techsupport/test_techsupport.py

```diff
@@ -266,10 +266,11 @@ def execute_command(duthost, since):
     :param duthost: DUT
     :param since: since string enterd by user
     """
-    stdout = duthost.command("show techsupport --since={}".format('"' + since + '"'))
-    if stdout['rc'] == SUCCESS_CODE:
-        pytest.tar_stdout = stdout['stdout']
-    return stdout['rc'] == SUCCESS_CODE
+    result = duthost.command("show techsupport -r --since={}".format('"' + since + '"'))
```

Comment:
If `rc` of the `command` module is not 0, then an exception will be raised. Need to add argument `module_ignore_errors=True` to avoid raising such exception:
```
    result = duthost.command("show techsupport -r --since={}".format('"' + since + '"'), module_ignore_errors=True)
```

Reply:
Thanks @wangxin - I've corrected the change according to your suggestion.